modules/configuring-localnet-switched-topology.adoc

```diff
@@ -47,6 +47,24 @@ spec:
 <3> The name for the additional network from which traffic is forwarded to the OVS bridge. This additional network must match the name of the `spec.config.name` field of the `NetworkAttachmentDefinition` CRD that defines the OVN-Kubernetes additional network.
 <4> The name of the OVS bridge on the node. This value is required only if you specify `state: present`.
 <5> The state for the mapping. Must be either `present` to add the bridge or `absent` to remove the bridge. The default value is `present`.
++
```

Comment:
With 4.19, we set up CUDN that configures the NAD as described by JSON below? Should CUDN and NAD relation highlighted here?

Reply:
Let us stick with NAD only here as I think Joe is working on CUDN with localnet topology.